mne/preprocessing/tests/test_maxwell.py

```diff
@@ -157,4 +163,66 @@ def test_maxwell_filter_additional():
     assert_equal(cov_raw_rank, raw.info['nchan'])
     assert_equal(cov_sss_rank, maxwell.get_num_moments(int_order, 0))
 
+
+@testing.requires_testing_data
+def test_maxwell_filter_fine_calibration():
+    """Test fine calibration feature of Maxwell filter
+
+    Notes
+    -----
+    Elekta MaxFilter uses single precision, so expect filtered results to
+    differ slightly.
+    """
+
+    # Load testing data (raw, SSS std origin, SSS non-standard origin)
+    with warnings.catch_warnings(record=True):  # maxshield
+        raw = Raw(raw_fname, preload=False, proj=False,
+                  allow_maxshield=True).crop(0., 1., False)
+    raw.preload_data()
+    raw.pick_types(meg=True, eeg=False)
+    with warnings.catch_warnings(record=True):  # maxshield, naming
+        sss_fine_cal = Raw(sss_fine_cal_fname, preload=True, proj=False,
+                           allow_maxshield=True)
+
+    # Create coils
+    all_coils, _, _, meg_info = _prep_meg_channels(raw.info, ignore_ref=True,
+                                                   elekta_defs=True)
+    picks = [raw.info['ch_names'].index(ch) for ch in [coil['chname']
+                                                       for coil in all_coils]]
+
+    # Test 1D SSS fine calibration
+    raw_sss = maxwell.maxwell_filter(raw, origin=[0., 0., 40.],
```

Comment:
to save some lines, do `from mne.preprocessing import maxwell_filter` at the top. Same with the private functions, you can do `from mne.preprocessing.maxwell import ...`. It's what we usually do anyway.